YDS-Storybook/SwiftUI/Atom/TextViewPageView.swift

```diff
@@ -0,0 +1,111 @@
+//
+//  TextViewPageView.swift
+//  YDS-Storybook
+//
+//  Created by 오동규 on 12/27/23.
+//
+
+import SwiftUI
+import YDS_SwiftUI
+
+struct TextViewPageView: View {
+    let title: String = "TextView"
+
+    @State var text: String = ""
+    @State var textColorSelectedIndex: Int = 0
+    @State var placeholderText: String? = "댓글을 입력해주세요."
+    @State var placeholderTextColorSelectedIndex: Int = 2
+    @State var multilineTextAlignmentSelectedIndex: Int = 0
+    @State var textViewTextIsEmpty: Bool = true
```

Comment:
```suggestion
    @State var isTextViewTextEmpty: Bool = true
```
사용하지 않는 State라면 제거 하면 좋을것 같아요~

Reply:
넵 알겠습니다! 수정해보도록 할게요